### PR TITLE
Initial RFC process with template

### PR DESCRIPTION
**Pull request by dragan**
- Documents our initial RFC process adapted from the Rust project
  - Includes a template for RFC authors to use when creating a new RFC

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+   closed. However, sometimes substantial new arguments or ideas are raised. 
+   When this happens, the RFC is canceled and the it goes back into development 
+   mode.
+
+Life Cycle
+----------
+
+Once an RFC reaches "active" status, an issue will be created in the
+`scipian/community` repository and the label `type/epic` will be applied. Using 
+the RFC as reference, the epic can then be broken down into it's story's. Since
```
Comment (nicklathe) [inline README.md]: `story's` should be `stories`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+we have multiple repository's in the Scipian organization, the story's should be
```
Comment (nicklathe) [inline README.md]: `repository's` should be `repositories`.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+created in the corresponding repository the story belongs to. When the work has 
+been broken down then the authors may implement it and follow the pull request 
+model to the appropriate Scipian repository's. Please refer to
```
Comment (nicklathe) [inline README.md]: `repositories`

File: README.md [hunk 4]
```diff
@@ -0,0 +1,270 @@
...
+
+Life Cycle
+----------
+
+Once an RFC reaches "active" status, an issue will be created in the
+`scipian/community` repository and the label `type/epic` will be applied. Using 
+the RFC as reference, the epic can then be broken down into it's story's. Since
+we have multiple repository's in the Scipian organization, the story's should be
+created in the corresponding repository the story belongs to. When the work has 
+been broken down then the authors may implement it and follow the pull request
```
Comment (nicklathe) [inline README.md]: There should be a comma after `down`, e.g. `When the work has been broken down, then ...`

File: README.md [hunk 5]
```diff
@@ -0,0 +1,270 @@
...
+[CONTRIBUTING][contributing] document for further instructions.
+
+Please keep in mind that once an "RFC" becomes "active", it does not mean that
+it's a rubber stamp and that the work will be merged. However, it does mean that
+in principle all the major stakeholders have agreed to the feature and are
+amendable to merging it.
+
+Nor does being "active" imply anything about what priority is assigned to it's
+implementation. Nor does it imply anything about whether a Scipian developer has
+been assigned the taks of implementing the feature. While it is not _necessary_
```
Comment (nicklathe) [inline README.md]: `task` is misspelled.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,270 @@
...
+that the author of the RFC also write the implementation, it is by the most
+effective way to see an RFC through to completion. Authors should also not
+expect that other project developers will take on responsibility of implementing
+their accepted feature.
+
+Modifications to "active" RFCs can be done in follow-up pull requests. We 
+strive to write each RFC in a manner that it will reflect the final design of 
+the feature; but the nature of the process means that we cannot expect every
```
Comment (nicklathe) [inline README.md]: Nit: `,` instead of `;`.

**Review by nicklathe — CHANGES_REQUESTED** [review]
Some small grammatical catches.

File: README.md [hunk 7]
```diff
@@ -0,0 +1,270 @@
...
+Before Creating an RFC
+----------------------
+
+Please do not try to rush the process, there are many reasons it is in place and
+one major reason is having a well thought out feature that is documented. This
+will allow implementing the feature much smoother, because you will know what 
+you are building before any work is done.
+
+Rushing the process will definitely hurt it's chances of acceptance. For example
+RFCs that are hastily, low quality, previously rejected, or those that don't fit
```
Comment (ncronquist) [inline README.md]: hastily written? hastily proposed? I think we need a verb for what is being done hastily to the RFC.

Comment (dragan) [reply README.md]: I liked `proposed`.

File: README.md [hunk 8]
```diff
@@ -0,0 +1,270 @@
...
+
+Please keep in mind that once an "RFC" becomes "active", it does not mean that
+it's a rubber stamp and that the work will be merged. However, it does mean that
+in principle all the major stakeholders have agreed to the feature and are
+amendable to merging it.
+
+Nor does being "active" imply anything about what priority is assigned to it's
+implementation. Nor does it imply anything about whether a Scipian developer has
+been assigned the task of implementing the feature. While it is not _necessary_
+that the author of the RFC also write the implementation, it is by the most
```
Comment (ncronquist) [inline README.md]: Either remove `by` or add `by far`.

**Review by ncronquist — COMMENTED** [review]
Sorry @dragan I got pulled away earlier and didn't get to finish going through this. I noticed just one other small missing word, but otherwise I think the content looks really good!

**Comment by dragan** [comment]
> Sorry @dragan I got pulled away earlier and didn't get to finish going through this. I noticed just one other small missing word, but otherwise I think the content looks really good!

No worries, thanks for taking the time to look over the PR.

**Review by nicklathe — DISMISSED** [review]
Squash commits then :shipit:

**Review by nicklathe — APPROVED** [review]
:shipit:

**Pull request merged**